### DISCUSSION TITLE
网卡ipv6问题

### DISCUSSION THREAD

**Discussion by tao517419** [jeessy2/ddns-go#940] [General]
![Screenshot_20231219_085122_edit_16899918226929](https://github.com/jeessy2/ddns-go/assets/85066690/4a1dc793-678f-4156-a430-d4e4a5e2142f)
![Screenshot_20231219_085110](https://github.com/jeessy2/ddns-go/assets/85066690/59534406-adae-4f2b-ab4b-058d581ace87)
每次刚开始可以获得v6（wlan0）过一段时间就获取不了，只能看到ccmni，就失败

**Comment by jeessy2** [comment]
改为通过接口获取

**Reply by tao517419** [reply]
我选择默认的接口，过一段时间也是失败，也不长久

**Reply by tao517419** [reply]
有没有其他更多的更好的接口

**Reply by jeessy2** [reply]
按照你说的，和接口无关
检查你网络

**Reply by tao517419** [reply]
好的，我知道了
